lib/state/selectors.ts

```diff
@@ -54,3 +55,8 @@ export const showTrash: S.Selector<boolean> = ({ ui: { collection } }) =>
   collection.type === 'trash';
 export const isDialogOpen = (state: S.State, name: T.DialogType['type']) =>
   state.ui.dialogs.find(({ type }) => type === name) !== undefined;
+
+export const numberOfNonEmailTags: S.Selector<number> = ({ data }) =>
+  [...data.tags.values()]
+    .filter((tag) => !isEmailTag(tag.name))
+    .map((tag) => tag.name).length;
```

Comment:
No need for this map - it’s redundant

Reply:
You are 1000% correct. See https://github.com/Automattic/simplenote-electron/pull/2723

Reply:
Also the selector may even be overkill. It’s fine as is, but I don’t see us needing to know often how many tags there are

Reply:
That's fair. I was actually going to make it return an array of `TagName`s (thus why I had that map in there) so it could be used elsewhere but then I read the comment 🤷‍♀️